core/code/entity_decode.js

```diff
@@ -73,7 +73,10 @@ function corePortalData(a) {
   }
 };
 
-var SUMMARY_PORTAL_DATA_LENGTH = 14;
+/* 
+history: bit_0 = visited, bit_1 = captured, bit_2 = scouted
+*/
+var SUMMARY_PORTAL_DATA_LENGTH = 19;
```

Comment:
May be worth defining this as a const as it doesn't appear to be re-assigned anywhere else.

Reply:
The code will be converted to a new ES version, but that is a different ISSUE.
I don't think we should start to mix in the source for the core.